charts/jobs/templates/cronjob.yaml

```diff
@@ -1,6 +1,6 @@
 {{- $v := .Values }}
 {{- range $name, $spec := $v.tasks }}
-{{- if and $spec.enabled (eq $spec.type "cronjob") }}
+{{- if and $spec.enabled (eq $spec.jobType "cronjob") }}
```

Comment:
Why, when we are in the context of jobs, can we not refer to "type" of a job (which to me obviously implies that it refers to a job)?

I actually like this implicitness as it is distinct at the same time. It can ONLY refer to the main subject in question: the job.

I dislike it when I am treated as deemed incapable of NOT making that distinction. The logic and linguistics are clear.

Same goes for giving an "id" prop to an entity. It does not need to duplicate it's shape in its id naming, when the shape itself is known. I actually think it's ugly and confusing.

Reply:
I agree, I just had a syntax problem in this code block (pseudo-code for ease):

```
definitions:
  job:
    type:
      type: string
```

I take it this is valid syntax? If so my modification from type to jobType isn't necessary.